Composer/packages/ui-plugins/virtual-assistant-creation/src/components/provisionSummaryPage.tsx

```diff
@@ -0,0 +1,131 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/** @jsx jsx */
+import { jsx } from '@emotion/core';
+import formatMessage from 'format-message';
+import React, { Fragment } from 'react';
+import { RouteComponentProps } from '@reach/router';
+import { DialogWrapper, DialogTypes } from '@bfc/ui-shared';
+
+import { Pivot, PivotItem } from 'office-ui-fabric-react/lib/Pivot';
+import { DialogFooterWrapper } from './dialogFooterWrapper';
+import { RouterPaths } from '../shared/constants';
+import { DetailsList, DetailsListLayoutMode, IColumn, SelectionMode } from 'office-ui-fabric-react/lib/DetailsList';
+import { TextField } from 'office-ui-fabric-react/lib/TextField';
```

Comment:
This should have been caught by the linter, but the import order here is incorrect.

Reply:
Locally I have it formatting on save but it applies 'formatting' based on the TS standard, not based on the lint rules in the Composer/eslintrc.js. Im not sure how to set it up to use the linting rules from Composer/eslintrc.js, let alone how to auto fix linting errors on format. Any tips on setting this up? This workspace follows the linting approach of other UI plugins so as I understand it all of them would have this issue.